docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: '2'
+services:
+  web:
+    build: .
+    command: npm start
+    ports:
+      - "4567:4567"
+    volumes:
+      - .:/braintree_express_example
+    environment:
```

Comment:
It's because of this line: https://github.com/braintree/braintree_express_example/blob/master/bin/www#L15

It defaults to port 3000 unless you set the env variable.